config/servicemappings/alloydb.yaml

```diff
@@ -164,6 +175,29 @@ spec:
             version: v1beta1
             kind: AlloyDBCluster
           parent: true
+        - tfField: instance_type
+          description: |-
+            (Required) The type of instance.
```

Comment:
Looks like you didn't run `make ready-pr` to update the description in the schema and elsewhere?

Reply:
Ran `make ready-pr`. Description is visible in schema